Stratis.Bitcoin/Features/MemoryPool/Fee/TxConfirmStats.cs

```diff
@@ -1,73 +1,102 @@
+using Microsoft.Extensions.Logging;
+using NBitcoin;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using Microsoft.Extensions.Logging;
-using NBitcoin;
 
 namespace Stratis.Bitcoin.Features.MemoryPool.Fee
 {
+    /// <summary>
+    /// Transation confirmation statistics.
+    /// </summary>
     public class TxConfirmStats
     {
+        /// <summary>Logger for logging messages for this object.</summary>
         private readonly ILogger logger;
 
-        // Sum the total feerate of all tx's in each bucket
-        // Track the historical moving average of this total over blocks
+        /// <summary>
+        /// Sum the total feerate of all tx's in each bucket.
```

Comment:
Looks more like a list of all feerate, rather than a sum.
Also tx's should be 'transactions' or 'txs'.